File: EngTaskGradingNetBESolution/EngTaskGradingNetBE/Models/Dtos/TeacherDto.cs
namespace EngTaskGradingNetBE.Models.Dtos
{
  public record TeacherLoginDto(
    string Email,
    string Password,
    bool RememberMe,
    string? CaptchaToken = null
  );

  public record TeacherCreateDto(
    string Email,
    string Password
  );
  public record TeacherDto(
    int Id,
    string Email
  );
}

File: eng-task-grading-react-fe/src/model/teacher-dto.ts
export interface TeacherRegisterDto {
  email: string;
  password: string;
}

export interface TeacherLoginDto {
  email: string;
  password: string;
  rememberMe: boolean;
captchaToken?: string;
}

export interface TeacherDto {
  id: number;
  email: string;
}